Formula/dotnet.rb

```diff
@@ -0,0 +1,77 @@
+class Dotnet < Formula
+  desc ".NET Core"
+  homepage "https://dotnet.microsoft.com/"
+  url "https://github.com/dotnet/source-build.git",
+      tag:      "v3.1.108-SDK",
+      revision: "05a0c8f6b13fcbec4b441c977b639b75e8a74f67"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on xcode: :build
+  depends_on "curl"
+  depends_on "icu4c"
+  depends_on "openssl"
+
+  # Patch of https://github.com/dotnet/source-build/pull/1789 which will be
+  # released with tag 3.1.110 in November 2020.
+  resource "0005-Fix-bad-configure-tests.patch" do
+    url "https://raw.githubusercontent.com/dotnet/source-build/17c6409189ed29f0fac2e8f4b1c30d882e6756b5/patches/coreclr/0005-Fix-bad-configure-tests.patch"
+    sha256 "57b83f9445d59137bdcc31c2a64d413bae23e80dc18f6fbcd8ceaac1d8b6754b"
+  end
+
+  def install
+    resource("0005-Fix-bad-configure-tests.patch").stage buildpath/"patches/coreclr/0005-Fix-bad-configure-tests.patch"
```

Comment:
According to the binlog, the build isn't finding this patch:

```
PatchesToApply
    /private/tmp/dotnet-20201013-85454-17gas77/patches/coreclr/0001-Allow-separate-tool-and-SDK-directories.patch
    /private/tmp/dotnet-20201013-85454-17gas77/patches/coreclr/0002-Conditionally-override-PackageVersion-properties.patch
    /private/tmp/dotnet-20201013-85454-17gas77/patches/coreclr/0003-Exclude-optdata-from-source-build.patch
    /private/tmp/dotnet-20201013-85454-17gas77/patches/coreclr/0004-Remove-Helix-Dependency.patch
```

Are you sure `stage` wants a file path rather than a dir? Can you check what ended up happening in `patches/coreclr`, if anything?

Reply:
You are quite correct, `patches/coreclr/0005-Fix-bad-configure-tests.patch` was created as a directory with the file `0005-Fix-bad-configure-tests.patch` within it. Thanks for noticing. Fixed!